Add files via upload

schweinsbergs opened the pull request:
Added first weekly reflection.

schweinsbergs committed:
Add files via upload

Added first weekly reflection.

alfarozavalae commented on Reflections/week_1_reflection.md:
```diff
@@ -0,0 +1,27 @@
+
+## What challenges or difficulties did you face? 
+
+Right now I'm having issues actually manipulating the dataframe, and comparing the ingredients data. The 
+reason for this is because it contains words like 'cups', 'tsp', etc. I'll have to consider stripping those 
+in the comparison. I didn't get it fixed this week, though I have reached out with a few students and plan 
+on going to TAs. 
```
Sounds good! 
You know when and were to find us (lab). Each TAs hours have been added to moodle in case you want to specifically meet with one of us.

alfarozavalae commented on Reflections/week_1_reflection.md:
```diff
@@ -0,0 +1,27 @@
+on going to TAs. 
+
+## What do you plan on accomplishing in the following week? 
+
+* Figuring out how to compare user's list against the ingredients list, even if it still has the 
+caveat of ingredient amount. 
+  
+* Calculating the percentage matched. 
+
+* Adding more data to compare against. 
+
```
These three goals look attainable but some other components of "S.M.A.R.T." goals are missing. For the next reflection, do not forget to add the with estimated number of hours/day to complete each goal.

alfarozavalae reviewed (approved):
Good job!
I believe from our conversation today that you have successfully figured out how to fix the bugs you were having and that you have also accomplished your three goals for the week already so you are already ahead. 
For next week, please remember to add your reflection and issue a PR for review. Also, remember to make your goals S.M.A.R.T.

schweinsbergs committed:
Create week_2_reflection

Added the week 2 reflection-- I still don't know why pycharm is trying to push blank files.

Commit:
Added week 3 reflection document.
